clients/rust/marginfi-cli/src/entrypoint.rs

```diff
@@ -114,6 +117,8 @@ pub enum GroupCommand {
         protocol_ir_fee: f64,
         #[clap(long, arg_enum)]
         risk_tier: RiskTierArg,
+        #[clap(long, arg_enum)]
+        oracle_type: OracleTypeArg,
```

Comment:
prob wanna have `pyth_oracle` changed to `oracle_key` as well here, since that name will appear in the CLI help?